<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdatePassword</class>
 <widget class="QDialog" name="UpdatePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Parol</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/img/icon/main.png</normaloff>:/images/img/icon/main.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
	background:rgba(0,0,0,0.4);
	border-radius:15px;
}

QFrame:hover{
	background:rgba(0,0,0,0.6);
}

QLineEdit{
	font-size:25px;;
	color:white;
	border:none;
	background:transparent;
	border-bottom:1px solid white;
}

QLineEdit:hover{
	background:rgba(0,0,0,0.04);
}


QPushButton{
	background:rgb(83, 152, 255);
	border-radius:10px;
	color:#fff;
}

QPushButton:hover{
	border:none;
	background:rgb(68, 117, 222)
}

QCheckBox{
	color:silver;
	background:transparent
}
QCheckBox:hover{
	color:white;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>431</width>
     <height>411</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="baslik_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:30px;
color:white;
background:transparent;
text-align: center;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>Parol almashtirish</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="old_password">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>351</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>51</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Eski parol</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Yangi parol</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Yangi parol qayta</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_updatepassword">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:rgb(83, 152, 255);
	border-radius:10px;
	color:#fff;
	font-size: 16px;
}

QPushButton:hover{
	border:none;
	background:rgb(68, 117, 222)
}
</string>
    </property>
    <property name="text">
     <string>Parolni o'zgartirish</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>451</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background-image: url(:/images/img/background/login_2.jpg);
	border-top-left-radius:0px;
	border-top-right-radius:0px;
	border-bottom-left-radius:25px;
	border-bottom-right-radius:25px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background:rgba(0,0,0,0.60);
}
QFrame{

border-radius:none;

}

QFrame:hover{
background:rgba(0,0,0,0.75);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>40</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgba(0, 0, 0,0.15);
color:silver;

border-radius:none;
}


QPushButton:hover{
background:red;
color:white
}</string>
    </property>
    <property name="text">
     <string>âœ•</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:silver;
background:transparent;</string>
    </property>
    <property name="text">
     <string>Parol</string>
    </property>
   </widget>
  </widget>
  <zorder>frame_3</zorder>
  <zorder>frame</zorder>
  <zorder>frame_2</zorder>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
